lib/logflare/backends/adaptor/postgres.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Logflare.Backends.Adaptor.Postgres do
```

Comment:
lets change this to `Adaptor.PostgresAdaptor` for clarity and shift the file to same dir as webhook_adaptor to match the elixir module placement conventions.